File: Python/01.Python-Basic-April-2019/13.nested_loops_exercise/09.password_generator.py
n = int(input())
l = int(input())

for i1 in range(1,n+1):
    for i2 in range(1,n+1):
        for i3 in range(int('a'),l+1):
            for i4 in range(int('a'),l+1):
                for i5 in range(2,n):
                    if i5>i1 and i5>i2:
                        print(f'{i1}{i2}{i3}{i4}{i5}')

File: JavaScript/02.JavaScript-Fundamentals-May-2021/17.textProcessing-Exercise/09.passwordGenerator.js
function solve(input) {
    let vowels = ['a', 'e', 'i', 'o', 'u'];
    let first = input.shift();
    let sec = input.shift();
    let word = input.shift();
    let con = first + sec;
    let start = 0;
    let output = '';

    for (const char of con) {
        if (vowels.includes(char)) {
            if (start == word.length) {
                start = 0;
            }
            output += word[start].toUpperCase();
            start++;
        } else {
            output += char;
        }
    }
    console.log(`Your generated password is ${output.split('').reverse().join('')}`)
}

solve([
    'ilovepizza',
    'ihatevegetables',
    'orange']
)